src/main/kotlin/ar/edu/unahur/obj2/vendedores/CentroDeDistribucion.kt

```diff
@@ -0,0 +1,24 @@
+package ar.edu.unahur.obj2.vendedores
+
+class CentroDeDistribucion {
+    val ciudad = mutableListOf<Ciudad>()
+    val vendedores = mutableListOf<Vendedor>()
+    var vendedoresFirmes = vendedores.filter { v -> v.esFirme() }
+    fun agregarVendedor(vendedor: Vendedor) {
+        if (vendedores.contains(vendedor)) {
+            throw Exception ("ya esta agregado")
+        } else {
+            vendedores.add(vendedor)
+        }
```

Comment:
Si se ejecuta la rama verdadera del `if`, el método explota y nunca llega al `else`. Podés entonces obviarlo, te quedaría así:
```suggestion
        if (vendedores.contains(vendedor)) {
            throw Exception ("ya esta agregado")
        }
        vendedores.add(vendedor)
```